utils/tests_fetcher.py

```diff
@@ -227,6 +239,8 @@ def create_reverse_dependency_map():
         something_changed = False
         for m in all_files:
             for d in direct_deps[m]:
+                if d not in direct_deps:
+                    raise ValueError(f"KeyError:{d}. From {m}")
```

Comment:
This helped me identify errors simpler during debugging, but shouldn't change anything at runtime. Let me know if you'd like for me to remove it and I will happily remove it @sgugger

Reply:
No it's good to have a clear error message. Maybe something even more detailed (even if we can read the traceback):
```
ValueError(f"KeyError when building the dependency map, {d} is not in {m}")
```